### PR TITLE
[CPM-168] add notes to plan

### PR DESCRIPTION
**Pull request by HongShi10**
## Changes

- Removed notes from course allocation
- Added notes for plans
## Checklist

- [x] Have tests been written?
- [x] Is the UI consistent?

## Requests

Addresses [CPM-168]

[CPM-168]: https://momentum-agile.atlassian.net/browse/CPM-168

### CODE REVIEW

File: api/src/models/plan.js [hunk 1]
```diff
@@ -75,6 +80,10 @@ const planSchema = mongoose.Schema({
         type: Boolean,
         required: true,
     },
+    notes: {
+        type: [String],
+        require: false
```
Comment (Dinith1) [inline api/src/models/plan.js:85]: shouldn't it be `required`, not `require`

**Review by Dinith1 — CHANGES_REQUESTED** [review]

File: api/src/models/plan.js (hunk 1 above)
Comment (HongShi10) [reply api/src/models/plan.js:85]: oh yikes forgot to push that change

**Review by Dinith1 — APPROVED** [review]
LGTM Boss :shipit:

**Review by wryao64 — APPROVED** [review]
LGTM

**Pull request merged**